charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -330,6 +330,10 @@ controller:
   dnsConfig: {}
 
 node:
+
+  # allows you to deploy aws-ebs-csi-node daemonset to separate namespace (make sure namespace exists before deloy) 
```

Comment:
Nit: Move this comment and parameter between lines 407-408 because it is less important than these top parameters.

Reply:
Hi, just moved this comment down, also defaulted namespaceOverride to blank string to match values file style